docs/examples/example_lime.ipynb

```diff
@@ -54,7 +54,7 @@
     "* {math}`f`, the original model -- in our case the MetaLearner\n",
     "* {math}`G`, the class of possible, interpretable surrogate models\n",
     "* {math}`\\Omega(g)`, a measure of complexity for {math}`g \\in G`\n",
-    "* {math}`\\pi_x(z)` a proximity measure of {math}`z` with respect to data point {math}`x`\n",
+    "* {math}`\\pi_x(z)` a proximity measure of an instance {math}`z` with respect to data point {math}`x`\n",
```

Comment:
Do you mean by "instance" a (potentially new) datapoint?

Reply:
Yes!